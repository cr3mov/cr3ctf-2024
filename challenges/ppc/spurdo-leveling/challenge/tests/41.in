189216325456951896 949135588339541446
3 1262
2 8085
