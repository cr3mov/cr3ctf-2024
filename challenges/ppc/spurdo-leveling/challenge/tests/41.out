81292176174860 81292176174860 81292176174860
